APMrover2/control_modes.cpp

```diff
@@ -73,46 +85,45 @@ void Rover::read_trim_switch()
     case CH7_DO_NOTHING:
         break;
     case CH7_SAVE_WP:
-		if (channel_learn->get_radio_in() > CH_7_PWM_TRIGGER) {
+        if (channel_learn->get_radio_in() > CH_7_PWM_TRIGGER) {
             // switch is engaged
-			ch7_flag = true;
-		} else { // switch is disengaged
-			if (ch7_flag) {
-				ch7_flag = false;
+            ch7_flag = true;
+        } else {  // switch is disengaged
+            if (ch7_flag) {
+                ch7_flag = false;
 
 				switch (control_mode) {
 				case MANUAL:
```

Comment:
I'm not sure what happened here but this is missing corrections and probably the reason why it failed CI checks.